src/vivarium/framework/results/manager.py

```diff
@@ -180,8 +180,6 @@ def register_observation(
         excluded_stratifications: List[str] = (),
         when: str = "collect_metrics",
```

Comment:
I still don't have the historical context here: are we defaulting to "collect_metrics" b/c that's simply the most common time to observe?

Reply:
I think so.